app/auth/profile/organisation/grade/[grade].tsx

```diff
@@ -7,11 +7,13 @@ import IconButton from "../../../../../components/IconButton";
 import useGrades, { GradeDTO } from "../../../../../hooks/useGrades";
 import { colors, ScaleSize, ScaleSizeH, ScaleSizeW, SharedStyles } from "../../../../../utils/SharedStyles";
 import SearchBar from "../../../../../components/SearchBar";
+import { useWeekplan } from "../../../../../providers/WeekplanProvider";
 
 const ViewGrade = () => {
+  const [searchedCitizens, setSearchedCitizens] = useState<string>("");
+  const { setIsCitizen, setId, id } = useWeekplan();
```

Comment:
Check warning on line 14 in app/auth/profile/organisation/grade/[grade].
'id' is assigned a value but never used